src/components/WwwFrame/TheHeader.vue

```diff
@@ -772,6 +791,10 @@ export default {
 			hasDepositBefore,
 		});
 		window.addEventListener('resize', this.determineIfMobile());
+
+		if (this.$refs?.newExpHeader) {
+			this.$refs?.newExpHeader?.getSuggestions?.(this.apollo);
```

Comment:
In general would it be simpler to pass apollo to the component then have to use the ref in the parent?